### PR TITLE
PSIMA-132: Updating initial condition functions

### PR DESCRIPTION
**Pull request by sourabhdalvi**
- This PR relies on PowerSystems #453 being merged, and adding `time_at_status` field to ThermalStandard

### CODE REVIEW

**Comment by jd-lara** [comment]
We need to merge https://github.com/NREL-SIIP/PowerSystems.jl/pull/454 to tag PSY@0.15.0 and test this.

**Comment by jd-lara** [comment]
bump to @sourabhdalvi what's the status of this PR?

**Comment by sourabhdalvi** [comment]
> bump to @sourabhdalvi what's the status of this PR?

should be passing now

File: src/core/update_initial_conditions.jl [hunk 1]
```diff
@@ -310,12 +310,12 @@ function _get_reservoir_energy_value(device, key)
     return PSY.get_initial_storage(device)
 end
 
-function _get_active_power_duration_value(dev, key)
+function _get_duration_value(dev, key)
     if key.ic_type == TimeDurationON
```
Comment (jd-lara) [inline src/core/update_initial_conditions.jl:314]: we don't need this conditional any more.

**Pull request merged**